docs/appendix/message-transport-mechanism.md

```diff
@@ -111,20 +123,10 @@ Assuming the Internet domain name of the remote participant is example.com, the
 
 [^B14]: P. Mockapetris, ""Domain names - concepts and facilities", STD 13, RFC 1034," 1987.
 
-The message will remain in the service discovery stage until all required data is available, or a participant-configurable and possibly message-class specific timeout timer expires.
-A DNS reply must have the NOERROR status, as well as syntactically valid and DNSSEC-authenticated content, in order to be considered usable.
-Other status codes such as SERVFAIL/NXDOMAIN, as well as connection failures or invalid record contents are considered temporary errors, in which case the lookup will be retried later within the timeout period.
-
 To prevent transient Internet connectivity or participant configuration issues from inhibiting message exchange, the timer value used for such a period should be at least one hour for routine messages.
 If service discovery fails due to inability to collect all required data within this timeout period, the delivery of the message fails.
 Such messages are removed from the outgoing queue and returned to the sending process, indicating a transport error.
 Note that, although message queue implementations may inform the sending process about delivery delays, only the final success or failure status is authoritative.
-When retrying failed requests, the implementation must implement an exponential back off mechanism to prevent undue load on the remote service.
-The implementation should set a reasonable maximum delay interval based on local factors, such as network capacity.
-
-For test situations or small-scale deployments where service discovery via DNS is too cumbersome, or requires unreasonable amounts of additional work, participants should be able to specify service information in a local configuration file.
-Since activities on the energy market are most likely regulated by  national oversight bodies, and such organizations are natural candidates for running the authoritative DNS zone for accredited market participants, implementations are encouraged to support a list of DNS suffixes that correspond to the zones maintained by such regulators, and to automatically query these zones.
-For example, if an energy.authority.test is on the DNS suffix list, this zone is queried for _usef.example.com.energy.authority.test in order to determine if example.com is an authorized market participant.
 
 <figure markdown>
   ![Example of the DNS relationships between a single USEF endpoint and multiple zones](../assets/images/image24.svg){ width=1000px }
```

Comment:
Personally, I would remove this detailed figure as well as it could imply the full DNS solution is part of the spec

Reply:
Removed